packages/nuxt/src/core/templates.ts

```diff
@@ -343,9 +343,19 @@ const inlineConfig = ${JSON.stringify(nuxt.options.appConfig, null, 2)}
 /** client **/
 // Vite - webpack is handled directly in #app/config
 if (import.meta.dev && !import.meta.nitro && import.meta.hot) {
-  const { updateAppConfig } = await import('#app/config')
+  const { reactive } = await import('vue')
```

Comment:
is there a reason you're using a dynamic import here from `vue`? I think you can directly import above and have it tree-shaken out in production mode...

Reply:
I tried to match my code's behavior to that of the original. I’d be happy to make it a static import if there are no side effects from this Vue import.